requre/storage.py

```diff
@@ -86,6 +87,71 @@ class DataTypes(Enum):
     Dict = 2
 
 
+class StorageKeyStrategy(Enum):
```

Comment:
I am not sure if we want to have `Enum` with all these methods.

Reply:
you are right, I've stared with ``Enum`` and now it is more than Enum :-)